packages/edit-post/src/components/sidebar/post-link/index.js

```diff
@@ -53,38 +53,46 @@ function PostLink( {
 			onToggle={ onTogglePanel }
 		>
 			{ isEditable && (
-				<TextControl
-					label={ __( 'URL' ) }
-					value={ forceEmptyField ? '' : currentSlug }
-					onChange={ ( newValue ) => {
-						editPermalink( newValue );
-						// When we delete the field the permalink gets
-						// reverted to the original value.
-						// The forceEmptyField logic allows the user to have
-						// the field temporarily empty while typing.
-						if ( ! newValue ) {
-							if ( ! forceEmptyField ) {
+				<div className="editor-post-link">
+					<TextControl
+						label={ __( 'URL Slug:' ) }
```

Comment:
Hey @timwright12 - It looks like this accidentally changed back to 'URL Slug' in this commit https://github.com/WordPress/gutenberg/pull/12959/commits/6049de64108fc46e2d89c623bdefa9ad3c4b26ab

I think there was an agreement on 'Page URL', so I thought I'd flag it.

Reply:
Oh wait, no, I see this comment now:
https://github.com/WordPress/gutenberg/pull/12959#issuecomment-456729023

Will give it a proper test and review in that case 😄